src/AutoDirectionProvider.jsx

```diff
@@ -22,12 +22,14 @@ function AutoDirectionProvider({
   direction,
   inline,
   text,
+  ...rest
 }) {
   const textDirection = getDirection(text);
   const dir = textDirection === 'neutral' ? direction : textDirection;
 
   return (
     <DirectionProvider
+      {...rest}
```

Comment:
we've generally tried to avoid doing this; spreading props is generally an antipattern :-/ there's no explicit propType warnings, so invalid DOM props end up only being runtime warnings.

This will also allow people to pass `style` and `className` props, which we've explicitly gone out of our way to avoid allowing, per https://brigade.engineering/don-t-pass-css-classes-between-components-e9f7ab192785

Reply:
Ah, that was my intention haha. I wanted to re-use this DOM node as a wrapper `<div>`.

I was kind of iffy about this PR to begin with, but I think you've reminded me of a good point. Thank you, I will close this PR.